Mergin/ui/ui_sync_dialog.ui

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>SyncDialog</class>
+ <widget class="QDialog" name="SyncDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>842</width>
+    <height>289</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Mergin - Synchronization</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="labelMergin">
+     <property name="pixmap">
+      <pixmap>../images/logo_color.svg</pixmap>
```

Comment:
can we make it smaller, like in config dialog, it seems to me huge now
![image](https://user-images.githubusercontent.com/23185498/76063761-a49a7b80-5f88-11ea-9ca0-8b5f8bc2a2ab.png)